pkg/models/user_token.go

```diff
@@ -23,11 +25,18 @@ type UserToken struct {
 	UnhashedToken string
 }
 
+type RevokeAuthTokenCmd struct {
+	AuthTokenId int64 `json:"authTokenId"`
+}
+
 // UserTokenService are used for generating and validating user tokens
 type UserTokenService interface {
```

Comment:
Perhaps this is two different services? One for auth/cookies and one for token management.

Reply:
Don't know. Everything feels very related in UserTokenService. Feels like an extra amount of code to split this up and have two different auth token services on HTTPServer struct plus the RevokeToken method will be shared between these two services.